### DISCUSSION TITLE
Change Signal edit menu to three dots icon

### DISCUSSION THREAD

**Discussion by zetic-be** [datalust/seq-tickets#2099] [Ideas]
Hello,

I have searched for a solution to this but I could not find it, so I am posting as a new feature idea.

It is possible to edit a Signal, but it seems like there is no option to remove them.
Could you please tell me how to do this or add this feature please?

Thank you

**Comment by liammclennan** [comment]
When editing a signal there is a drop-down menu in the top right corner, which has a 'delete' option. 

<img width="312" alt="image" src="https://github.com/datalust/seq-tickets/assets/271514/4fe4e7e2-66b5-427f-a770-43e919d99f79">

**Comment by zetic-be** [comment]
Perfect, thank you.
I would suggest to change the icon to a vertical three dots.  
This one is generaly used for collapsible panels.
So I transform this idea with this suggestion.

Best regards
